app/helpers/enterprises_helper.rb

```diff
@@ -88,4 +88,11 @@ def shop_trial_expired?(enterprise)
   def remaining_trial_days(enterprise)
     distance_of_time_in_words(Time.zone.now, enterprise.shop_trial_start_date + Spree::Config[:shop_trial_length_days].days)
   end
+
+  # Enable embeddeding within iframes on external sites.
+  # Call with :after_filter on controller actions.
+  def enable_embedded_shopfront
```

Comment:
I'll take a look at maybe a spec to test if the http header has been properly changed, but for the most part I don't think we can test this in a spec. It really needs an iframe in a page in a different domain to load the page remotely, and I don't think we can do that with Rspec?